multi-agent/tests/unit/scripts/test_migrate_builtin_reverse_no_key.py

```
@@ -0,0 +1,57 @@
+    with pytest.raises(RuntimeError, match="no --encryption-key was provided"):
+        _run_reverse_migration_body(client, "TestDB", dry_run=False, cipher=None)
+
+
+def test_reverse_migration_passes_plaintext_fields_without_key():
+    from run.scripts.migrate_builtin_system import _run_reverse_migration_body
+
+    overlay_fields = {"model_name": "gpt-4"}
+    db = _make_mock_db(overlay_fields)
+
+    client = MagicMock()
+    client.__getitem__ = lambda self, key: db
```

Ruff (0.16.0)

[warning] Unused lambda argument: `self`

(ARG005)